docker-compose.test.yml

```diff
@@ -5,6 +5,7 @@ services:
     build:
       context: ./tests
       dockerfile: Dockerfile
+    command: tail -f /dev/null
```

Comment:
could you add a comment explaining why you do that?

Reply:
yes